repo/packages/B/bookkeeper/1/marathon.json.mustache

```diff
@@ -8,26 +8,41 @@
     "type": "DOCKER",
     "volumes": [
       {
-        "containerPath": "/bk/journal",
-        "hostPath": "/mnt/journal",
+        "containerPath": "/bookkeeper/data/journal",
+        "hostPath": "journal",
         "mode": "RW"
       },
       {
-        "containerPath": "/bk/index",
-        "hostPath": "/mnt/index",
+        "containerPath": "journal",
+        "mode": "RW",
+        "persistent": {
+          "size": {{bookkeeper.journal_volume_size}}
+        }
+      },
+
+      {
+        "containerPath": "/bookkeeper/data/index",
+        "hostPath": "index",
         "mode": "RW"
       },
       {
-        "containerPath": "/bk/ledgers",
-        "hostPath": "/mnt/ledgers",
+        "containerPath": "index",
+        "mode": "RW",
+        "persistent": {
+          "size": {{bookkeeper.index_volume_size}}
+        }
+      },
+
+      {
+        "containerPath": "/bookkeeper/data/ledgers",
```

Comment:
shouldn't this be parameterized instead of hard coded?

Reply:
Thanks, will change it.